src/index.tsx

```diff
@@ -1,6 +1,9 @@
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 
+React.createElement('div');
```

Comment:
why do you need that ?
In my opinion it's redundant

Reply:
It throws an error without importing React or if I import React but don't call it.

Reply:
![image| {width:100%}](https://github.com/mate-academy/react_dynamic-list-of-goods/assets/52879984/7b783417-f478-4e66-8bfa-701160762e2a)

From different task, but same error.